changelogs/fragments/snmp_idempotancy_fix.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - ios_snmp_server - fixed facts not rendering user authentication password information in snmp_server.
```

Comment:
I'm curious about how just config changes results in a facts fix. Can you please elaborate? 

Also, a more general suggestion, for patches like this, it would help if you add more details in the PR summary including what changed, an example of how it currently behaves, as well as one that show how it would behave with the patch merged. This gives more context on the problem that you're trying to solve and facilitates more efficient reviews. :-)

Reply:
ahh this is a mistake, will change that